src/GraphQL.SystemTextJson/SchemaExtensions.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Text.Json;
+using System.Threading.Tasks;
+using GraphQL.Types;
+
+namespace GraphQL.SystemTextJson
+{
+    public static class SchemaExtensions
+    {
+        public static Task<string> ExecuteAsync(this ISchema schema, Action<ExecutionOptions> configure)
+        {
+            var documentWriter = new DocumentWriter(new JsonSerializerOptions() { WriteIndented = true });
```

Comment:
```suggestion
            var documentWriter = new DocumentWriter(new JsonSerializerOptions { WriteIndented = true });
```

Reply:
This will be deleted when I commit next.